L1Trigger/L1THGCal/src/HGCalTriggerTools.cc

```diff
@@ -42,17 +40,12 @@ void HGCalTriggerTools::eventSetup(const edm::EventSetup& es) {
   if (geom_->isWithNoseGeometry())
     noseLayers_ = geom_->noseTopology().dddConstants().layers(true);
 
-  if (geom_->isV9Geometry()) {
-    bhLayers_ = geom_->hscTopology().dddConstants().layers(true);
-    totalLayers_ = eeLayers_ + fhLayers_;
-  } else {
-    bhLayers_ = geom_->bhTopology().dddConstants()->getMaxDepth(1);
-    totalLayers_ = eeLayers_ + fhLayers_ + bhLayers_;
-  }
+  bhLayers_ = geom_->hscTopology().dddConstants().layers(true);
+  totalLayers_ = eeLayers_ + fhLayers_;
```

Comment:
Is the definition of totalLayers_ changed here?

Reply:
The definition of `totalLayers_` as `totalLayers_ = eeLayers_ + fhLayers_ + bhLayers_` was only for geometries <=V8, where the HCAL layers were divided into front and back sections. In geometries >= V9, the silicon section of HCAL is covering all the HCAL layers (given by `fhLayers_`). 
Before there was a switch to check whether the geometry was <=V8 or >=V9; now the switch has been removed. So the definition of `totalLayers_` for geometries >=V9 has not changed.